src/python/analyzer_executor/src/analyzer_executor_lib/analyzer_executor.py

```diff
@@ -122,24 +111,6 @@ def check_caches(
     return False
 
 
-def handle_result_graphs(analyzer, result_graphs, sender):
```

Comment:
so yeah fun fact
a) `vulture` is a great lib to detect unused code
b) these are unused

Reply:
sweet

Reply:
Ahhh, oh yeah, these used to be helpers for an optimization that I have sadly disabled.